src/stores/tenant.rs

```diff
@@ -435,4 +452,8 @@ impl TenantStore for DefaultTenantStore {
     ) -> Result<Tenant> {
         panic!("Shouldn't have run in single tenant mode")
     }
+
+    async fn suspend_tenant(&self, id: &str, reason: &str) -> Result<()> {
+        panic!("Shouldn't have run in single tenant mode")
```

Comment:
Error message could be improved:

```suggestion
        panic!("Invalid APNS or FCM credentials. Cannot suspend tenant in single-tenant mode, so panicking instead.")
```

Reply:
There error message is the same for all the functions, they cannot be run in single-tenant mode

Reply:
Maybe the other functions' error messages could be improved too?

I just don't see any debugging info that would help a service maintainer know that their credential had expired.

Reply:
wdym? The panics here are for function calls that won't/can't be done in single-tenant mode

Reply:
Ah, if they are impossible to call, then you should use [`unreachable!()`](https://doc.rust-lang.org/std/macro.unreachable.html) instead of `panic!()`